cmd/buildctl/debug/logs.go

```diff
@@ -30,7 +29,7 @@ var LogsCommand = cli.Command{
 func logs(clicontext *cli.Context) error {
 	args := clicontext.Args()
 	if len(args) == 0 {
-		return fmt.Errorf("build ref must be specified")
+		return errors.Errorf("build ref must be specified")
```

Comment:
Is `fmt.Errorf` deprecated?

Reply:
Btw `errors.New` seems suffice

Reply:
https://github.com/moby/buildkit/pull/3305